docs/docs/learn/introduction/gateway/index.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 2
+sidebar_label: "BOB Gateway: Bitcoin Intents"
+---
+
+# BOB Gateway: Bitcoin Intents
+
+## Introduction
+
+Bitcoin users can easily onboard to the BOB Hybrid L2 without previously holding any Ethereum assets. This page explains the structure of _BOB Gateway_, an intent-based bridge that coordinates peer-to-peer swaps between users and liquidity providers (LPs). Cross-chain transfers are secured by verifying Bitcoin transaction proofs with an on-chain light client, avoiding the need for an oracle. Optional intents, such as staking, lending, and swapping a small amount of ETH for transaction fees can all be accomplished while only requiring a single Bitcoin transaction from the user.
+
+## How Gateway Works
+
+1. Liquidity providers (LPs) temporarily lock wrapped Bitcoin (WBTC or tBTC) in escrow smart contracts on BOB.
+1. A user makes a request to the (trusted) relayer to reserve some of the available liquidity.
+1. The user sends BTC to the liquidity provider's Bitcoin address. A hash of the user's order is included in the `OP_RETURN` of their transaction, including data such as the recipient's EVM address on BOB.
+1. The relayer trustlessly verifies the user's Bitcoin transaction by submitting a merkle proof to an on-chain [Light Client](/learn/builder-guides/relay.md), granting the relayer permission to withdraw the LP's wrapped Bitcoin without needing to use an oracle.
+1. Gateway sends the LP's wrapped Bitcoin to the user's EVM address. If the user requested a Bitcoin LST/LRT, that token is minted using the LP's wrapped Bitcoin _before_ it is sent to the user.
+
+## Architecture
+
+![architecture](./architecture.png)
+
+### User Flow
+
+1. User requests to swap BTC for e.g. tBTC/WBTC or stake BTC for e.g., SolvBTC.BBN
+2. User gets "quote" which gateway contract to use
+3. User creates "order" with relayer based on available liquidity
+4. User creates tx and updates order with txid - should be done before publishing tx to avoid conflicts
+5. Relayer monitors Bitcoin chain and executes swap when txid seen
+
+Any frontend can integrate with Gateway by following the [BOB docs](https://docs.gobob.xyz/learn/builder-guides/gateway).
+
+### Liquidity Provider (LP) Flow
+
+1. LP asks relayer to deploy a gateway contract (permissioned at the moment because BOB pays for fees).
```

Comment:
```suggestion
1. LP asks relayer to deploy a gateway contract (permissioned at the moment because BOB pays for gas).
```

Reply:
I'm of the opinion that Bitcoin-facing writing should say "fees" instead of "gas" for familiarity. I would even go as far as to extend this opinion to the BOB Bridge/Stake UI, but that's out of my scope.

Reply:
Let's keep it consistent with what we have on the UI for now. If we change it, we need to change it in both places.